doc/man3/EVP_MAC.pod

```diff
@@ -235,7 +235,7 @@ Note that not all algorithms may support all digests.
 HMAC does not support variable output length digests such as SHAKE128
 or SHAKE256.
 
-=item B<OSSL_MAC_PARAM_SIZE> ("size") <unsigned integer>
```

Comment:
I think this one should stay _unsigned int_.  It is talking about the type of the PARAM not the C type.

Reply:
There are others above this that use integer...

Reply:
It is a size_t in other files, so I am going to leave it as a size_t.

Reply:
It would be better to change the size_t in those other files.

Reply:
I really dont agree, and I think this is VERY confusing :)
If this is supposed to be what type the param is then its a size_t - not a unsigned integer. 
We are using octet string etc which corresponds to the type we use when we construct one of these params. 
We dont do construct_uint() for it.

Reply:
The parameter data type is OSSL_PARAM_UNSIGNED_INTEGER.  As long as that type as well as the data size is compatible, it doesn't really matter of the one end uses uint and the other uses size_t.

The `OSSL_PARAM_` functions are pickier, true, but they are not mandatory (or do you want them to be?)

Reply:
I am thinking of the end user - not how you have implemented it under the hood.

It should be clear to a user that you need to call it this way..
size_t sz;
params[0] = OSSL_PARAM_construct_size_t(OSSL_MAC_PARAM_SIZE, &sz);

Reply:
If that is not how we should call it then we need to get rid of all accessors that use size_t :)

Reply:
We possibly need to discuss this disconnect -- I deliberated worded the man3 KDF pages in terms of the OSSL_PARAM types and the man7 pages in terms of the C types.  I don't think this is ideal.

Could we call it an _unsigned integer_ but add the expected maximum size in the description?
I.e. _The size of this parameter should not exceed that of B<size_t>._

Reply:
Please have a look at the discussion in #9875 and the resulting #9891